src/frontend/post-github-setup.ts

```diff
@@ -41,11 +41,7 @@ const validateJiraSite = (
 		.then((response) => {
 			// If Jira site is valid, response returns a state of RUNNING
 			if (response?.data?.state === "RUNNING") {
-				res.redirect(
-					req.session.githubToken
-						? getJiraMarketplaceUrl(jiraSiteUrl)
-						: "/github/login"
-				);
+				res.redirect(getJiraMarketplaceUrl(jiraSiteUrl));
```

Comment:
Am I correctly understand that now this redirect will happen in token validation middleware?

Reply:
Correct, but not for this route at it's not needed at all for the github setup page/post, hence the removal.